app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= stylesheet_link_tag 'signup', media: 'all', 'data-turbolinks-track': 'reload' %>
+
+<div class="login-container">
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { class: "login-form" }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+    <%= f.text_field :name, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up", class:"sign" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] [REQUIRED] Hello Team, you have done a great job so far with the devise implementation, however, the signup page does not presently look professional, so let's ensure it's organized and adjusted to look like the wireframe provided taking into consideration the width of the button and input field.

- I commend your efforts with the nice styles you added 👍🏽 

### Your Implementation 
<img width="1010" alt="Screenshot 2023-09-07 at 22 17 38" src="https://github.com/AleWaweru/blog-App/assets/58692590/9c0ff019-ec43-4c40-8fad-f0f05b9ba6a6">
### Wireframe Provided
<img width="382" alt="Screenshot 2023-09-07 at 22 25 02" src="https://github.com/AleWaweru/blog-App/assets/58692590/34a4f01f-ea54-4540-988f-8e6b7dddafd7">